.editorconfig

```diff
@@ -14,10 +14,34 @@ indent_size = 2
 [*.py]
 indent_size = 4
 
-# Don't correct spacing for sql and yaml files as sometimes want them wrong for tests
-[*.sql]
+# Don't correct indentation for sql and yaml files as sometimes want them wrong for tests
+[*.{yml,yaml,sql}]
+indent_style = unset
+
+# Some specific tests with trailing newlines
+# If adding any exceptions here, make sure to add them to .pre-commit-config.yaml as well
+[test/fixtures/linter/sqlfluffignore/*/*.sql]
+indent_style = unset
+trim_trailing_whitespace = unset
```

Comment:
I wonder if this should be unset more broadly? With some tests adding trailing whitespace is useful

Reply:
My personal preference would be to have trailing white space tests in Python strings (or quote YAML like in https://github.com/sqlfluff/sqlfluff/blob/main/test/fixtures/rules/std_rule_cases/L001.yml). Having them as actual trailing whitespace is not great as they are invisible to the human eye.

Reply:
But willing to let these few exceptions pass for now btw. And maybe add some more in future if we really have to. But would prefer them to very much be the exception.

WDYT?

Reply:
Happy with that - it was just a thought 👍